Server/src/Models/Assembler.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+
+// Schema
+const assembler_schema = mongoose.Schema({
```

Comment:
Rename to `schedule_schema`. adjust exports accordingly.

Reply:
changed